source/customization.rst

```diff
@@ -630,3 +630,34 @@ If the balance is defined as a ``project`` balance, then it applies to a project
      "project": "project1",
      "value": 10
    }
+
+Maintenance Mode
+-----------------
+
+As an administrator you may want to have some downtime of the Open OnDemand service for various reasons,
+while still telling your customers that the downtime is expected.
+
+You can do this by setting Open OnDemand in 'Maintenance Mode'. Apache will serve 
+``/var/www/ood/public/maintenance/index.html`` which you can change or brand to be your own. Changes
+to this file will persist through upgrades.
+
+Apache return this html file and a 503 response code to all users who's ip does not match one of the
```

Comment:
`s/Apache return/Apache returns/g` , plural returns.  Also probably good to capitalize `ip` to be `IP`.